week-1/interpret/time-format.js

```diff
@@ -13,17 +13,17 @@ console.log(result);
 
 // For the piece of code above, read the code and then answer the following questions
 
-// a) How many variable declarations are there in this program?
+// a) How many variable declarations are there in this program? - just 7
 
-// b) How many function calls are there?
+// b) How many function calls are there? - 2 
```

Comment:
This isn't quite right - what are the 2 function calls you're seeing?

Reply:
theres only 1 which is console.log